airflow-core/docs/redirects.txt

```diff
@@ -85,7 +85,7 @@ scheduler.rst administration-and-deployment/scheduler.rst
 # Installation
 installation.rst installation/index.rst
 upgrade-check.rst installation/upgrade-check.rst
-upgrading-to-2.rst howto/upgrading-from-1-10/index.rst
+howto/upgrading-from-1-10/index.rst upgrading-to-2.rst
```

Comment:
We can just nuke this. I don't we have a `upgrading-to-2.rst` anymore

Reply:
Yeah irrelevant.